packages/grid/default.env

```diff
@@ -71,7 +71,8 @@ S3_ROOT_PWD="admin" # needs randomizing
 S3_REGION="us-east-1"
  #not-using
 S3_PRESIGNED_TIMEOUT_SECS=1800
-S3_VOLUME_SIZE_MB=1024
+S3_VOLUME_SIZE_MB=40000
```

Comment:
Careful, this will allocate that space, which the system might not have.

We might need to add a flag to hagrid to make this a little clearer for people deploying without kubernetes.

In the helm chart its `s3VolumeSizeMB` and is currently defaulted to 1000 as well.